### PR TITLE
misc

### PR DESCRIPTION
**Pull request by emptyrivers**

### CODE REVIEW

File: WeakAuras/Types.lua [hunk 1]
```diff
@@ -1589,8 +1597,6 @@ WeakAuras.internal_fields = {
   parent = true,
   internalVersion = true,
   regionType = true,
```
Comment (InfusOnWoW) [inline WeakAuras/Types.lua]: regionType is probably also problemeatic.

Comment (emptyrivers) [reply WeakAuras/Types.lua]: Ah, you're probably right on that one. I think Display would be the best category for regionType, given just how much of the display tab data is governed by the regionType.

Comment (InfusOnWoW) [reply WeakAuras/Types.lua]: I agree

Comment (emptyrivers) [reply WeakAuras/Types.lua]: on further thought, sortHybridTable (and much of the dynamic group options such as growth direction) should be categorized as arrangement

File: WeakAuras/Types.lua [hunk 2]
```diff
@@ -1538,7 +1538,7 @@ WeakAuras.update_categories = {
   },
   {
     name = "display",
-    fields = {},
+    fields = {"regionType"},
```
[the remark this answers is not in the document]
Comment (emptyrivers) [reply WeakAuras/Types.lua]: True, I don't need to, but if it's alright with you I think I'll leave it.

**Pull request merged**